client/js/views/Home.vue

```diff
@@ -159,6 +149,25 @@ h1 {
   -webkit-text-fill-color: transparent;
```

Comment:
Moved css around for media query to min-width vs Max-width so that css for the larger screens are inside the media query. 

![image](https://user-images.githubusercontent.com/47369872/196721036-84155fbb-798b-493a-92df-1c550db90d38.png)

![image](https://user-images.githubusercontent.com/47369872/196721134-41942f15-b85f-42b4-818d-200fff9a968a.png)